actioncable/lib/rails/generators/js/channel_generator.rb

```diff
@@ -0,0 +1,7 @@
+module Js
+  module Generators
+    class ChannelGenerator < ::Rails::Generators::NamedBase
```

Comment:
I think it has to do with Thor command lookup, but I can't remember since it was a while I dug into them.